### ISSUE TITLE
Make LSTM checking modular

### ISSUE THREAD

**Issue by jusinowicz** [jusinowicz/yahara_flashiness#1]
Currently, updateModelLSTM and updateModelLSTM check to see if all LSTM models are current, and runs based on the ALL == T condition. 

Change this to check each model individually to make it more convenient to fix models that have become loss = NAN.
